src/components/App/index.tsx

```diff
@@ -60,8 +62,13 @@ export default class App extends Component<Props, State> {
       this.componentDidUpdate = (props, state) => {
         if (oldCDU) oldCDU.call(this, props, state);
         window.STATE = this.state;
+
+        // change title once component updated
+        this.changeTitle();
```

Comment:
This is inside `if (process.env.NODE_ENV === 'development') {` so it won't work in production.